bin/cyhy-nvdsync

```diff
@@ -43,15 +43,24 @@ def parse_json(db, json_stream):
 
     for entry in data.get("CVE_Items", []):
         cve_id = entry["cve"]["CVE_data_meta"]["ID"]
-        if "baseMetricV2" not in entry["impact"]:
-            # NVD 'reject' CVEs do not have 'baseMetricV2' CVSS data
+        if ("baseMetricV2" or "baseMetricV3") not in entry["impact"]:
+            # Reject CVEs that don't have baseMetricV2 or baseMetricV3 CVSS data
             # Make sure they are removed from our db.
             db.CVEDoc.collection.remove({"_id": cve_id}, safe=False)
             print "x",
         else:
             print ".",
-            cvss_base_score = entry["impact"]["baseMetricV2"]["cvssV2"]["baseScore"]
-            entry_doc = db.CVEDoc({"_id": cve_id, "cvss_score": float(cvss_base_score)})
+            if "baseMetricV3" in entry["impact"]:
+                cvss_base_score = entry["impact"]["baseMetricV3"]["cvssV3"]["baseScore"]
+                cvss_version = entry["impact"]["baseMetricV3"]["cvssV3"]["version"]
+            else:
+                cvss_base_score = entry["impact"]["baseMetricV2"]["cvssV2"]["baseScore"]
+                cvss_version = entry["impact"]["baseMetricV2"]["cvssV2"]["version"]
```

Comment:
I personally think this is a bit cleaner:
```suggestion
            baseMetric = "baseMetricV3" in entry["impact"] ? "baseMetricV3" : "baseMetricV2"
            cvss_base_score = entry["impact"][baseMetric]["cvssV3"]["baseScore"]
            cvss_version = entry["impact"][baseMetric]["cvssV3"]["version"]
```

Reply:
I like what you're thinking, but that doesn't quite work; note the `["cvssV3"]` vs. `["cvssV2"]` after the `[baseMetric]`.  How about this instead?
```suggestion
            version = "baseMetricV3" in entry["impact"] ? "V3" : "V2"
            cvss_base_score = entry["impact"]["baseMetric" + version]["cvss" + version]["baseScore"]
            cvss_version = entry["impact"]["baseMetric" + version]["cvss" + version]["version"]
```

Reply:
I did edit my suggestion to include the `"cvssV2"` vs `"cvssV3"` jazz quite some time ago, but I like what you did above even better.

Reply:
Ah, I totally missed your earlier edit, sorry!  Thanks for the assist here.

Reply:
Also, glad I tested this, because we both forgot that Python doesn't have ternary operator syntax like `<test> ? <if-true> : <if-false>`.  Please take a look at https://github.com/cisagov/cyhy-core/pull/69/commits/d5e722e2a9ca630064e02d7d6a5c8f354c98538d.

Reply:
Perfect!